lighthouse-core/lib/network-request.js

```diff
@@ -341,10 +370,77 @@ module.exports = class NetworkRequest {
     // Bail if we somehow already have transfer size data.
```

Comment:
this is probably where we got the wrong comment from last time if we want to fix it while we're in here :)

Reply:
(to clarify, absolutely no need to do anything here just noting it) :)

Reply:
Fixed 😉